Mathlib/CategoryTheory/Sums/Basic.lean

```diff
@@ -136,15 +144,13 @@ theorem swap_map_inr {X Y : D} {f : inr X ⟶ inr Y} : (swap C D).map f = f :=
 
 namespace Swap
 
-/- Porting note: had to manually call `cases f` for `f : PEmpty` -/
-
 /-- `swap` gives an equivalence between `C ⊕ D` and `D ⊕ C`. -/
 def equivalence : Sum C D ≌ Sum D C :=
   Equivalence.mk (swap C D) (swap D C)
     (NatIso.ofComponents (fun X => eqToIso (by cases X <;> rfl))
-      (by simp only [swap]; aesop_cat_nonterminal; cases f; cases f))
+      (by aesop_cat (add norm simp swap)))
```

Comment:
You can just drop this argument. `aesop_cat` can do it unaided, and it is the autoparam.

Reply:
Done x2.